### ISSUE TITLE
Feature request: Sort files alphanumerically

### ISSUE THREAD

**Issue by vitlav** [webtorrent/webtorrent-desktop#1185]
WebTorrent 0.18.0

There is no order in torrent file list, downloaded parts can be show like
part3.avi
part2.avi
part4.avi
part1.avi

I expected alphanumeric sort.

**Comment by bnjmnt4n** [comment]
@feross mentioned in https://github.com/webtorrent/webtorrent-desktop/issues/1076#issuecomment-264748538:

> We used to sort file names alphabetically, but we don't do it anymore. Instead, we just show the files in the order that the torrent file lists them. It's up to the torrent file creator to put the files in a sensical order.

**Comment by dsernst** [comment]
I want to kindly add the the current UI on this is not intuitive and makes handling large groups of files very difficult. This isn't exactly a fringe user experience. It's clearly come up a number of times — e.g. the linked issues marked as dupes.

In particular, I have a 10gb torrent w/ 222 files, but only want to download a single one of them. It's possible to `Pause All` and then re-select just the needed file, but very difficult to find in such a large list if sorted by filesize.

This has come up for me a handful of times. The current lack of support for **Sort by Name** requires keeping a different torrent program, exclusively for this use-case. Seems... excessive.

-----

I do see the linked `WONTFIX` comment from @feross. 

And these from PR https://github.com/webtorrent/webtorrent-desktop/pull/1188:

![image](https://user-images.githubusercontent.com/6340841/57163256-8af73900-6da5-11e9-87f7-cfba04279eda.png)

-----

I certainly admire & agree with the sentiment to avoid creating a bloated Settings page. Yuck.

But maybe there's a simpler UI to handle this? What about a single context menu option on the list of files: "Sort by Name".

No addition to the Settings page is needed.
The control for this would be right where someone uses it. 
And not visible to anyone who doesn't right-click the file list.

-----

Current context menu:

<img alt="Screen Shot 2019-05-03 at 1 58 17 PM" src="https://user-images.githubusercontent.com/6340841/57165348-a36a5200-6dab-11e9-8fe2-21eed5dc11e7.png">

Proposal:

```
Remove From List
Remove Data File
-----
Show in Finder
-----
Copy Magnet Link to Clipboard
Copy Instant.io Link to Clipboard
Save Torrent File As...
-----
Sort by Name
```

Selecting `Sort by Name` would mark it checked, similar to Finder.app:

![image](https://user-images.githubusercontent.com/6340841/57165880-36f05280-6dad-11e9-990f-4c226b500858.png)

-----

Happy to write up a PR for this. But don't want to waste anyone's time if it's unwelcome.

**Comment by dsernst** [comment]
I went ahead and coded this up (https://github.com/webtorrent/webtorrent-desktop/compare/master...dsernst:feature/sort-file-alphanumerically-1185): 

![sort-by-name](https://user-images.githubusercontent.com/6340841/57189768-7f039800-6ec7-11e9-81d2-94b525eba607.gif)

**Comment by ldexterldesign** [comment]
The more sort methods the merrier, until then back to http://qbittorrent.org for (not streaming) torrent management

Yours hopefully

**Comment by dsernst** [comment]
This can be closed now that #1582 is merged 🙂

**Comment by alxhotel** [comment]
Yep. Feature complete in 77317aa.
